client/src/extension.ts

```diff
@@ -156,7 +163,7 @@ function associateRshFiles() {
 }
 
 function injectRshFileAssocation() {
-	const settingsFile:string = `${rootFolder}${path.sep}.vscode/settings.json`;
+	const settingsFile:string = WORKSPACE_SETTINGS_PATH;
```

Comment:
Just realized this constant uses `rootFolder` before initialization. I would keep ``${rootFolder}${path.sep}.vscode/settings.json`` inlined.

Reply:
This has been all been updated in the latest commit